src/utils/route-validator.ts

```diff
@@ -0,0 +1,24 @@
+import { FormGroup } from '@angular/forms';
+
+export function RouteValid(controlName: string): any {
```

Comment:
https://angular.io/guide/form-validation#adding-custom-validators-to-template-driven-forms
Do it properly